uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/other/OtherScreen.kt

```diff
@@ -176,6 +153,39 @@ private fun AppBar(
     }
 }
 
+@Composable
+private fun Content(
```

Comment:
I think it depends on your taste, but it seems that it will take time to know which part is content, so how about BackdropFrontLayerContent?

Reply:
You can merge without changing. 👍

Reply:
Good idea! I'll change name.